spec/requests/users_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+RSpec.describe 'Posts', type: :request do
```

Comment:
- [ ] Great job working on this milestone. I think since you are writing specs tests for users_spec, you should have `RSpec.describe 'Users' `rather. Kindly review this this.

Reply:
@t-yanick 
thanks for your good and valuable feedback